// @spartan-ng/cli/src/generators/base/versions.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.TAILWIND_ANIMATE_VERSION = exports.TAILWINDCSS_VERSION = exports.TAILWIND_MERGE_VERSION = exports.SPARTAN_TOGGLE_VERSION = exports.SPARTAN_TABLE_VERSION = exports.SPARTAN_TABS_VERSION = exports.SPARTAN_SWITCH_VERSION = exports.SPARTAN_SHEET_BRAIN_VERSION = exports.SPARTAN_SEPARATOR_BRAIN_VERSION = exports.SPARTAN_SELECT_BRAIN_VERSION = exports.SPARTAN_RADIO_GROUP_BRAIN_VERSION = exports.SPARTAN_PROGRESS_BRAIN_VERSION = exports.SPARTAN_POPOVER_BRAIN_VERSION = exports.SPARTAN_MENU_BRAIN_VERSION = exports.SPARTAN_LABEL_BRAIN_VERSION = exports.SPARTAN_HOVERCARD_BRAIN_VERSION = exports.SPARTAN_FORMFIELD_VERSION = exports.SPARTAN_DIALOG_BRAIN_VERSION = exports.SPARTAN_CORE_VERSION = exports.SPARTAN_COLLAPSIBLE_BRAIN_VERSION = exports.SPARTAN_COMMAND_BRAIN_VERSION = exports.SPARTAN_AVATAR_BRAIN_VERSION = exports.SPARTAN_ALERT_DIALOG_BRAIN_VERSION = exports.SPARTAN_ACCORDION_BRAIN_VERSION = exports.NG_ICONS_VERSION = exports.FALLBACK_ANGULAR_VERSION = void 0;
// angular dependency
exports.FALLBACK_ANGULAR_VERSION = '^18.0.0';
// ng-icon dependency
exports.NG_ICONS_VERSION = '^29.10.0';
// spartan dependencies
exports.SPARTAN_ACCORDION_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_ALERT_DIALOG_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_AVATAR_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_COMMAND_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_COLLAPSIBLE_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_CORE_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_DIALOG_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_FORMFIELD_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_HOVERCARD_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_LABEL_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_MENU_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_POPOVER_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_PROGRESS_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_RADIO_GROUP_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_SELECT_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_SEPARATOR_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_SHEET_BRAIN_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_SWITCH_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_TABS_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_TABLE_VERSION = '0.0.1-alpha.380';
exports.SPARTAN_TOGGLE_VERSION = '0.0.1-alpha.380';
// dev dependencies
exports.TAILWIND_MERGE_VERSION = '^2.2.0';
exports.TAILWINDCSS_VERSION = '^3.0.2';
exports.TAILWIND_ANIMATE_VERSION = '^1.0.6';
